plugins/scaffolder/src/next/OngoingTask/TaskSteps/StepTime.test.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { StepTime } from './StepTime';
+import { render, act } from '@testing-library/react';
+
+describe('StepTime', () => {
+  it('should return empty when there is no start time', () => {
+    const step = {
+      id: 'test',
+      startedAt: undefined,
+      endedAt: undefined,
+      name: 'test',
+    };
+
+    const { container } = render(<StepTime step={step} />);
+
+    expect(container.querySelector('span')?.textContent).toBe('');
```

Comment:
just a nudge that `screen` is a great way of doing these queries in general, so you don't ever have to care about the return values from `render`.